pkgs/by-name/ht/http-nu/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  llvmPackages,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

Use `finalAttrs` pattern.